src/pages/scripts/tutorials/migrate-from-extendscript/index.md

```diff
@@ -0,0 +1,138 @@
+<!-- 
+TODO Q: Are these applicable to plugins as well?
+-->
+
+# ExtendScript to UXP Script
```

Comment:
@anoopvalomkot  Could this be applicable for UXP plugin developers as well?

Reply:
@padmkris123 
yes they are applicable, scripting DOM is common for both UXP scripting and UXP plugins

Reply:
My question is pertaining to ExtendScript concepts. Is that what you mean by InDesign Scripting DOM?

Reply:
yes, InDesign DOM APIs are same in ExtendScript, UXP Script and UXP plugins. There are minor variations in UXP w.r.t to ExtendScript, the same is captured in migration guide. This is applicable for UXP plugin too.